iodata/test/test_orca.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+# IODATA is an input and output module for quantum chemistry.
+#
+# Copyright (C) 2011-2019 The IODATA Development Team
+#
+# This file is part of IODATA.
+#
+# IODATA is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 3
+# of the License, or (at your option) any later version.
+#
+# IODATA is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, see <http://www.gnu.org/licenses/>
+#
+# --
+# pragma pylint: disable=invalid-name,no-member
+"""Test iodata.orca module."""
+
+import numpy as np
+
+from ..iodata import IOData
+from ..utils import angstrom
+try:
+    from importlib_resources import path
+except ImportError:
+    from importlib.resources import path
+
+
+def test_load_water_number():
+    # test if IOData has atomic numbers
+    with path('iodata.test.data', 'water_orca.out') as fn_xyz:
+        mol = IOData.from_file(str(fn_xyz))
+    check_water(mol)
+
+
+def test_load_water_element():
+    # test if IOData has atomic symbols
+    with path('iodata.test.data', 'water_orca.out') as fn_xyz:
+        mol = IOData.from_file(str(fn_xyz))
+    check_water(mol)
+
+
+def test_load_scf_energy():
+    # test if IOData has the correct energy
+    with path('iodata.test.data', 'water_orca.out') as fn_xyz:
+        mol = IOData.from_file(str(fn_xyz))
+    check_water(mol)
+
+
+def check_water(mol):
+    """Checks if atomic numbers and coordinates obtained from orca out file are correct.
+
+        Parameters
+        ----------
+        mol : IOData
+            IOdata dictionary.
+
+        Returns
+        -------
+        None
+    """
+    assert mol.numbers[0] == 8
+    assert mol.numbers[1] == 1
+    assert mol.numbers[2] == 1
```

Comment:
Again a very minor suggestion, mainly sharing it because you might like the trick:

```python
# The following line does not start with assert, because this is done inside the assert_equal function.
np.testing.assert_equal(mol.numbers, [8, 1, 1])
```

The module `np.testing` contains a lot of convenient tricks to write assertions for arrays. See https://docs.scipy.org/doc/numpy-1.13.0/reference/routines.testing.html

Reply:
Changed to np.testing. This is a nice feature. I will have a look.